tasks/task_database.c

```diff
@@ -804,9 +804,26 @@ static int database_info_list_iterate_found_match(
       entry.last_played_second= 0;
 
       playlist_push(playlist, &entry);
-      RARCH_LOG("[Scanner]: Add \"%s\"\n", entry_label);
+      RARCH_LOG("[Scanner]: Add \"%s\" to \"%s\"\n", entry_label, entry.db_name);
       if (retroarch_override_setting_is_set(RARCH_OVERRIDE_SETTING_DATABASE_SCAN, NULL))
-         printf("Add \"%s\"\n", entry.label);
+      {
+         /* Colorize " Add " prefix with green. */
+#ifdef _WIN32
+         HANDLE con = GetStdHandle(STD_OUTPUT_HANDLE);
+         if (con != INVALID_HANDLE_VALUE)
+         {
+            SetConsoleTextAttribute(con, 2);
+            WriteConsole(con, " Add ", 5, NULL, NULL);
+            SetConsoleTextAttribute(con, 7);
+         }
+         else
+            fputs(" Add ", stdout);
+#else
+         fputs("\x1B[32m Add \x1B[0m", stdout);
+#endif
```

Comment:
While I appreciate the colouring, I think it's adding code complexity over a simple `RARCH_LOG()`.

Reply:
Hmm, this console output is separate from actual logging (which is also done), and colors are useless when logging to a file, so not sure how complex things ought to get..

Reply:
Hmm, i see :thinking: